artifacts_previous/v3/contracts/payment/SessionsManager.sol/SessionsManager.sol

```diff
@@ -0,0 +1,437 @@
+/**
```

Comment:
@NicolasMahe try to verify the contracts using the `solcInputs` file, it seems to contains the source code